README.md

```diff
@@ -107,3 +107,13 @@ docker compose up -d
 ```shell
 npm run start:dev
 ```
+
+7. Before opening a PR, remember to check for vulnerabitilies and address them before merging :pray:
+
+```shell
+npm run audit 
+```
+
+- To get more information, and then `npm audit fix --force` to attempt resolution.
```

Comment:
The goal of "Shifting Left" Security is that vulnerability management _should be_ job 0 of any development workflow. Someone who just implemented a feature should run npm audit and confirm that the dependencies upon which his/her new feature depends, have not known vulnerabilitties, that perhaps a day ago didn't exist. So this should be as natural and as frequent as `npm install` or `npm test`, definitely before opening a PR. Dependabot is a good tool (and probably will have the same info that `npm audit`), but unfortunately usually not in the fore of any developers attention. Security should be in developers continuous software lifecycle since the beginning.

Reply:
An alternative, but a more intrusive one, is to set in Github that no PR can be merged if Dependabot has issues with it. But IMHO that's too intrusive and doesn't let the developers, and PR reviewers, used best criteria when to let a vulnerability pass as tech debt for a while... but again, the ideal is to make it a daily routine rather than an afterthought hidden in an obscure tab in github that is usually not controlled?

Reply:
Developers are not adding new dependencies daily. This is optimizing for an edge case, which is rarely a good idea.

And even when they add a new dependency, that does not happen lightly either. We have Architecture Forum and Tech Radar. We don't need to introduce artificial friction into daily routine of our engineers for something that brings no tangible value.

I'm OK with having an audit step as a part of CI workflow and failing build if there are runtime dependencies with vulnerabilities of severity higher than X - but I'm not sure if `better-npm-audit` supports that.